src/docs/admin/references/properties/scheduler.properties

```diff
@@ -43,7 +43,7 @@ pa.scheduler.home=.
 # Example https://myserver.mydomain.com/catalog
 #pa.catalog.rest.public.url=
 
-# Set this property to define the public url of the cloud automation service (e.g. if behind a reverse proxy or accessed through a domain)
+# Set this property to define the public url of the service automation (e.g. if behind a reverse proxy or accessed through a domain)
```

Comment:
the service automation -> Service Automation
(without the) so the reader understands that it  is about the name of the component.